plugin-dev/Source/Sentry/Private/GenericPlatform/GenericPlatformSentrySubsystem.h

```diff
@@ -56,6 +57,7 @@ class FGenericPlatformSentrySubsystem : public ISentrySubsystem
 
 	USentryBeforeSendHandler* GetBeforeSendHandler();
 	USentryBeforeBreadcrumbHandler* GetBeforeBreadcrumbHandler();
+	USentryBeforeLogHandler* GetBeforeLogHandler();
```

Comment:
```suggestion
	USentryBeforeLogHandler* GetBeforeLogHandler() const;
```

And ideally the others would be `const` too, but that's out-of-scope for this change